src/components/Patient/DailyRounds.tsx

```diff
@@ -790,36 +795,6 @@ export const DailyRounds = (props: any) => {
                 ]}
               />
 
-              <TextFormField
```

Comment:
@AdityaJ2305 The Spo2 field was changed to be a number field because there is no need for it to be a rangeautocomplete field. Can you instead keep that version?

Reply:
Sure